Content.Shared/Radiation/SharedRadiationStorm.cs

```diff
@@ -30,6 +31,7 @@ public abstract class SharedRadiationPulseComponent : Component
     [Serializable, NetSerializable]
     public class RadiationPulseState : ComponentState
     {
+        public readonly float RadsPerSecondBase;
         public readonly float RadsPerSecond;
```

Comment:
Just adding this in here is not sufficient to make sure that this variable is networked.

You also need to add it to the constructor (see how `RadsPerSecond` is set a few lines below this), and you also need to then make use of it in `HandleComponentState()` in the client's `RadiationPulseComponent`.

Alternatively, if it just doesn't need to be networked (which seems to be the case, given that its only used to set `RadsPerSecond` which is networked), then you can just remove it from the state.

Reply:
And also remove the dirtying when setting.

I.e., the code from before:
```cs
public override float RadsPerSecondBase
{
    get => _radsPerSecond;
    set
    {
        _radsPerSecond = value;
        Dirty();
    }
}
```
could just be
```cs
public override float RadsPerSecondBase { get; set; }
```